vespaclient-container-plugin/src/main/java/com/yahoo/vespa/http/server/FeedHandlerV3.java

```diff
@@ -43,6 +43,9 @@
     protected final ReplyHandler feedReplyHandler;
     private final Metric metric;
     private final Object monitor = new Object();
+    private int remainingThreadsForFeedingAllowance;
+    private final long msBetweenBumpingMaxThreads;
+    private long nextTimeToAllocateAnotherThread;
```

Comment:
Minor: I would have used Duration and Instant for these two

Reply:
Thanks, now I will use that for time in java from now on.